app/views/admin/pre_auth_states/index.html.erb

```diff
@@ -41,6 +45,30 @@
         </div>
       <% end %>
     </div>
+  <% end.empty? %>
+    <i>There are no users in an unverified state</i>
+  <% end %>
+
+  <hr>
+  <h5>New Flow Users</h5>
```

Comment:
They're not really "users" or "contacts"—I'd say they're "emails." Fyi, I think I would've listed @unverified_users = User.where(state: 'unverified')` and then for each of those `user.email_address.first.confirmation_pin `, etc. because `@unverified_contacts = ContactInfo.`... on the other hand, will also list additional emails that users have added to their account after the fact (after signup) but have not gotten around to verifying yet.

Reply:
What if they contact CS about one of those 'other emails' they added? (although they should only be adding one during initial signup.


I did `@unverified_users = User.where(state: 'unverified')` first.. but it ended up with me having to do more loops to get all the unverified emails.

Reply:
> What if they contact CS about one of those 'other emails' they added? 

Well, that's a different story, that's what the security logs and User Search page are for. This page is to answer the question "how many people started the signup process but didn't finish?" That's exactly what a PreAuthState is.

Reply:
I thought this page was to give CS access to the users' pin if they don't get the email.